### ISSUE TITLE
Support for tracing from content child process

### ISSUE THREAD

**Issue by janodvarko** [firebug/firebug.next#241]
It should be possible to use FBTrace.sysout from within a child process.

It's currently possible to get instance of FBTrace global and all it's method from a child process, but there is no tracing in the tracing console.

Honza

**Comment by janodvarko** [comment]
Patch committed at
https://github.com/firebug/tracing-console/commit/c5520e210996f5d743bf6c3ae21e038d99e60ad4

Honza

**Comment by janodvarko** [comment]
Example:

```
// sendSyncMessage is available in a child process
let scope = {};
Cu["import"]("resource://fbtrace/firebug-trace-service.js", scope);
var FBTrace = scope.traceConsoleService.getTracer("extensions.firebug", sendSyncMessage);
```

**Comment by janodvarko** [comment]
$fixed firebug-3.0.0-alpha.7
